dto/api_input.py

```diff
@@ -67,38 +66,53 @@ class TeamsLeagueDTO(BaseDTO):
 
 class TeamsStatisticsOrPlayersDTO(BaseDTO):
     sport_id = fields.Int(required=False, missing=None)
+    sport = fields.Str(required=False, missing=None)
     team_id = fields.Int(required=False, missing=None)
     league_id = fields.Int(required=False, missing=None)
-
+    name = fields.Str(required=False, missing=None)
+    page = fields.Int(required=False, missing=0)
+    per_page = fields.Int(required=False, missing=0)
```

Comment:
We shouldn't make our TeamsDTO do Pagination work. It would violate SOLID (even though ever so slightly).  Since our paginations works the same way for every entity we have (I would like to hope so at least) we could play with some decorators/interfaces etc etc. 

In C# I would write an Interface for paginations - not sure what the best approach for Python would be, so you'll have to search for it yourself